src/pudl/transform/eia.py

```diff
@@ -543,24 +537,34 @@ def harvesting(  # noqa: C901
     mcs = consistency["consistent_ratio"].mean()
     logger.info(f"Average consistency of static {entity} values is {mcs:.2%}")
 
-    if entity == "plants":
-        entity_df = _add_additional_epacems_plants(entity_df)
-        entity_df = _add_timezone(entity_df)
+    return entity_df, annual_df, col_dfs
 
-    eia_transformed_dfs[f"{entity}_annual_eia"] = annual_df
-    entities_dfs[f"{entity}_entity_eia"] = entity_df
-    if debug:
-        return entities_dfs, eia_transformed_dfs, col_dfs
-    return (entities_dfs, eia_transformed_dfs)
 
-
-# TODO (bendnorman): Can this and harvesting() be separate assets?
-def _boiler_generator_assn(  # noqa: C901
-    eia_transformed_dfs: dict[str, pd.DataFrame],
-    eia923_years: list[int] | None = None,
-    eia860_years: list[int] | None = None,
-    debug: bool = False,
-) -> dict[str, pd.DataFrame]:
+@asset(
+    ins={
+        table_name: AssetIn()
+        for table_name in [
+            "clean_boiler_generator_assn_eia860",
+            "clean_generation_eia923",
+            "clean_generators_eia860",
+            "clean_boiler_fuel_eia923",
+        ]
+    },
```

Comment:
is this `ins` being read into `boiler_generator_assn_eia860` as `eia_transformed_dfs`? it might be more clear if we call `eia_transformed_dfs` something more like `clean_boiler_generator_inputs` or something less relic-y of our old pass around all of the transformed dfs

Reply:
We can call it whatever we like, or we can actually enumerate the dataframes as independent inputs, which would be more explicit. I think I did this in order to minimize the need to touch the insides of the BGA function.

Reply:
Switch name to `clean_*_dfs` here / and in other assets like this.